app/views/pages/home/_section_partners.html.erb

```diff
@@ -4,12 +4,12 @@
       <h3 class="fr-ml-2v fr-col-12"><%= t('.title').html_safe %></h3>
       <p class="fr-ml-2v fr-col-12"><%= t('.subtitle') %></p>
 
-      <div class="providers center">
+      <div class="providers">
         <% @providers.each do |provider| %>
-        <div class="card-provider fr-m-2v fr-pt-2w fr-pr-2w fr-pl-2w">
-          <%= image_tag("providers/#{provider.slug}.png") %>
-          <h5 class="fr-pt-1v fr-pb-1v"><%= provider.name %></h5>
-        </div>
+          <a href="<%= provider.link_catalogue_filtered_by %>" target="_blank" class="card-provider fr-m-2v fr-pt-2w fr-pr-2w fr-pl-2w">
```

Comment:
Pourquoi y'a un target blank ici ?

Reply:
Me semble qu'on veut ouvrir la doc dans une nouvelle page si on est depuis la homepage

Reply:
Hum ok, étrange mais why not.